### PR TITLE
Add templates

### PR DESCRIPTION
**Pull request by EvanMcLaughlin-eaton**

### CODE REVIEW

File: .github/workflows/pr-comment.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: ubuntu-latest
+    name: comment PR if needed
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v1
+        with:
+           message: 'Thank you for the master pull request. 👍 Please be sure this pull request is for the master branch. The master branch is used for items that are ready for publishing.
+           The dev branch is our working branch for all issues and enhancements. Please read our [Contributing Guidelines](https://github.com/brightlayer-ui/.github/blob/dev/CONTRIBUTING.md) for more information.'
```
Comment (EvanMcLaughlin-eaton) [inline .github/workflows/pr-comment.yml]: Just realized we're going to need a contributing guide somewhere.

Comment (huayunh) [reply .github/workflows/pr-comment.yml]: I added those markdown files to the docit repo. Once these changes [go to the master branch](https://github.com/etn-ccis/blui-doc-it/pull/621), you can confidently do [something like this](https://github.com/etn-ccis/blui-react-component-library/pull/678/files#diff-782e917f57ecc86d636cfe3a672f92f80b5ab6946bddd72ebae06209576e49beR18)

Comment (jvgreiner) [reply .github/workflows/pr-comment.yml]: so will the Contributing Guidelines link point to doc'it?
Contributing Guidelines](https://github.com/brightlayer-ui/.github/blob/dev/CONTRIBUTING.md) for more information.'

Comment (huayunh) [reply .github/workflows/pr-comment.yml]: it will be pointing to somewhere in the docit repo, but not the docit website

**Review by huayunh — APPROVED** [review]

**Pull request merged**